packages/@ember/polyfills/lib/assign.ts

```diff
@@ -28,6 +30,20 @@ export function assign(target: object, ...sources: any[]): any;
   @static
 */
 export function assign(target: object) {
+  deprecate(
+    'Use of `assign` has been deprecated. Please use `Object.assign` or the spread operator instead.',
+    false,
+    {
+      id: 'ember-polyfills.deprecate-assign',
+      until: '5.0.0',
+      url: 'https://deprecations.emberjs.com/v4.x/#toc_ember-polyfills-deprecate-assign',
```

Comment:
Need to make a PR to the deprecation app, and land it so that we can merge this (once we confirm the URL works)

Reply:
Way ahead of you 😉 

https://github.com/ember-learn/deprecation-app/pull/899